source/src/MonitorElementManager.cc

```diff
@@ -230,7 +230,13 @@ namespace dqm4hep {
 
       const bool objectStat(TObject::GetObjectStat());
       TObject::SetObjectStat(false);
-      TObject *pTObject = pTFile->Get(fullName.getPath().c_str());
+      TObject *pTObject = nullptr;
+      // FIXME : bad handling of path in root path
```

Comment:
Is it still a problem with the way you do it now? I don't see why

Reply:
Problem is when you use "/" or not to get an object in a root file. At some point I should refactor this in a proper function...